Tee'sApp/Controllers/ViewController.swift

```diff
@@ -0,0 +1,186 @@
+//
+//  ViewController.swift
+//  Tee'sApp
+//
+//  Created by La Pommeray, Thalia on 10/25/22.
+//
+
+import UIKit
+
+class ViewController: UIViewController, UITableViewDataSource{
+    
+    @IBOutlet weak var tableView: UITableView!
+        
+    @IBOutlet weak var searchField: UITextField!
+    
+    @IBOutlet weak var apiActivityIndicator: UIActivityIndicatorView!
+    
+    
+    var data = ["butterfly", "dog", "cat", "mouse", "rabbit", "bat", "chimpanzee", "donkey",
+                "dinosaur", "duck", "eagle", "goldfish"]
+    let activityIndicator = UIActivityIndicatorView()
+    var userName:String = ""
+    var viewModel = InfoViewModel()
```

Comment:
`viewModel` can be a private constant (`private let`) since it's never modified or accessed outside this VC